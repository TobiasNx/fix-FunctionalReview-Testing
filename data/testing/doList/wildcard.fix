set_array("coll[]")
do list(path:"feld?", "var":"$i")
    add_field("coll[].$append.feldtest","true")
    copy_field("$i.a.value", "coll[].$last.a")
end
retain("coll[]")
