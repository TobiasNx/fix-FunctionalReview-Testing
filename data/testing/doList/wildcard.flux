FLUX_DIR + "wildcard_test.xml"
|open-file
|decode-xml
|handle-generic-xml
|fix( FLUX_DIR + "wildcard.fix" )
|encode-json(prettyPrinting="true")
| write(FLUX_DIR + "wildcard_result.json")
;
