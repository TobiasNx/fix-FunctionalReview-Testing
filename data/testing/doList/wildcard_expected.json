{
  "coll" : [ {
    "feldtest" : "true",
    "a" : "Dog"
  },{
    "feldtest" : "true",
    "a" : "Ape"
  },{
    "feldtest" : "true",
    "a" : "Giraffe"
  }, {
    "feldtest" : "true",
    "a" : "Crocodile"
  },
 {
    "feldtest" : "true",
    "a" : "Human"
  },
 {
    "feldtest" : "true",
    "a" : "Bird"
  }
 ]
}
